packages/edit-site/src/components/layout/index.js

```diff
@@ -296,82 +281,82 @@ export default function Layout() {
 
 					<SavePanel />
 
-					{ showCanvas && (
-						<>
-							{ isListPage && <PageMain /> }
```

Comment:
The list page is now rendered within the "content" area while the editor is rendered within the "preview" area.